src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_defaults.py

```diff
@@ -1,8 +1,8 @@
 '''
 This module holds the customization settings for the debugger.
 '''
-from _pydevd_bundle.pydevd_constants import QUOTED_LINE_PROTOCOL
+from _pydevd_bundle.pydevd_constants import HTTP_JSON_PROTOCOL
 
 
 class PydevdCustomization(object):
-    DEFAULT_PROTOCOL = QUOTED_LINE_PROTOCOL
+    DEFAULT_PROTOCOL = HTTP_JSON_PROTOCOL
```

Comment:
This should be fixed by https://github.com/microsoft/ptvsd/pull/1881 (in review).